deployment-detector/README.md

```diff
@@ -1,26 +1,24 @@
-# Large Tether Transfer Agent
+# Challenge-1
 
 ## Description
-
-This agent detects transactions with large Tether transfers
+This  monitors bot deployments from the Nethermind deployer address (0x88dC3a2284FA62e0027d6D6B1fCfDd2141a143b8)
 
 ## Supported Chains
-
-- Ethereum
-- List any other chains this agent can support e.g. BSC
+- Polygon
 
 ## Alerts
 
 Describe each of the type of alerts fired by this agent
 
-- FORTA-1
-  - Fired when a transaction contains a Tether transfer over 10,000 USDT
-  - Severity is always set to "low" (mention any conditions where it could be something else)
-  - Type is always set to "info" (mention any conditions where it could be something else)
-  - Mention any other type of metadata fields included with this alert
+- Nethermind Forta-Bot-Deployer Detector
+  - Fired when Nethermind contract at : 0x88dc3a2284fa62e0027d6d6b1fcfdd2141a143b8 deploys an agent
+  - Severity is always set to "info" 
+  - Type is always set to "info" 
+  - Metadata fields:
+     - agentId: agentId of the deployed agent
+     - metadata: ipfs of the metadat of the agent
+     - chainsIds: list of networks supported 
```

Comment:
Since `metadata` is not longer being included in your bot's metadata, this needs to be updated. Also, `owner` seems to have been left off.

Lastly, the metadata properties in the README are usually put in ticks (`), so they show up like so: `agentId`, `owner`, and `chainIds`.

Oh, and "chain**s**Ids" has a small spelling mistake